src/components/block-button.js

```diff
@@ -17,12 +17,20 @@ AFRAME.registerComponent("block-button", {
   },
 
   block(clientId) {
-    NAF.connection.adapter.block(clientId);
+    if (NAF.connection && NAF.connection.adapter) {
+      NAF.connection.adapter.block(clientId);
+    }
   },
 
   // Currently unused
   unblock(clientId) {
-    NAF.connection.adapter.unblock(clientId);
-    NAF.connection.entities.completeSync(clientId);
+    if (NAF.connection) {
```

Comment:
do you know if we can expect `NAF.connection` to always be a truthy property?

Reply:
Yes, `NAF.connection` is always a `NetworkConnection` object, it's defined immediately when you reference `networked-aframe`